daemon/k8s_watcher.go

```diff
@@ -474,9 +474,7 @@ func (d *Daemon) EnableK8sWatcher(reSyncPeriod time.Duration) error {
 					return nil
 				}
 			},
-			func(m versioned.Map) versioned.Map {
-				return m
-			},
+			d.missingK8sEndpointsV1,
```

Comment:
I guess this is the crux of the matter here, instead of considering all of the endpoints in `m` missing, we now actually check which ones are missing.

Reply:
Correct.